superdesk/__init__.py

```diff
@@ -13,7 +13,7 @@
 import blinker
 import logging as logging_lib
 from flask import abort, json, Blueprint, current_app as app  # noqa
-from flask.ext.script import Command as BaseCommand, Option  # noqa @UnresolvedImport
```

Comment:
I think here you should keep the `Option`, but remove `@UnresolvedImport` comment
and that would be the case for the other lines too